### PR TITLE
[tripy] Batchnorm implementation from feature branch

### PR DESCRIPTION
**Pull request by farazkh80**
Batchnorm implementation, currently be used in ResNet50 notebook PR, adding it as a general module similar to `LayerNorm`.

Example
```python
batch_norm = tp.BatchNorm(3)

input = tp.iota((2, 3, 1, 1))
output = batch_norm(input)
```

Tested the code and documnation.

### CODE REVIEW

**Comment by farazkh80** [comment]
moved from fork to feature branch @pranavm-nvidia

File: tripy/tests/integration/test_batchnorm.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import torch
```
Comment (pranavm-nvidia) [inline tripy/tests/integration/test_batchnorm.py:1]: Do you have the pre-commit hook enabled? This file should have had a copyright header added.

**Review by pranavm-nvidia — APPROVED** [review]

File: tripy/tests/integration/test_batchnorm.py (hunk 1 above)
Comment (farazkh80) [reply tripy/tests/integration/test_batchnorm.py:15]: fixed now, thanks!
